### ISSUE TITLE
Add usage examples to documenation

### ISSUE THREAD

**Issue by Seluj78** [Seluj78/flask-utils#11]
Add usage examples in the `usage` section of the documentation

**Comment by Seluj78** [comment]
If you're still looking for stuff to do @Mews, I'm certain this is something you could do with ease seeing as how good you've handled anything so far 🙌

**Comment by Mews** [comment]
Actually I'm not at all familiar with using flask, I've just been contributing to other things that don't directly involve it.
So I don't know how I'd write usage examples for the library if I don't know how to use it myself.

**Comment by Seluj78** [comment]
Fair enough, just wanted to know :) I'll do it anyway when I get some time !
